addons/l10n_it_edi/models/res_company.py

```diff
@@ -153,3 +156,15 @@ def _l10n_it_edi_export_check(self):
                 'action': self.env['res.config.settings']._get_records_action(name=_("Settings"), context=new_context),
             }
         return errors
+
+    @api.depends('l10n_it_has_tax_representative')
+    def _compute_l10n_it_tax_representative_partner_id(self):
+        for company in self:
+            if not company.l10n_it_has_tax_representative:
+                company.l10n_it_tax_representative_partner_id = False
+
+    @api.depends('l10n_it_tax_representative_partner_id')
+    def _inverse_l10n_it_tax_representative_partner_id(self):
+        for company in self:
+            # Clear or set box, based off of partner
+            company.l10n_it_has_tax_representative = bool(company.l10n_it_tax_representative_partner_id)
```

Comment:
```suggestion
        for company in self:
            company.l10n_it_has_tax_representative = bool(company.l10n_it_tax_representative_partner_id)
```
The comment doesn't add any information